packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -204,7 +204,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, any> {
    */
   private _validateProps(props: IBreadcrumbProps): void {
     const { items, overflowIndex } = props;
-    if (overflowIndex! < 0 || overflowIndex! > items.length - 1) {
+    if (overflowIndex! < 0 || overflowIndex! > items.length) {
```

Comment:
This fixes the issue when items is of 0 length. However, if items is of length N and overflowIndex is also N, I think it will consider that a valid scenario when it isn't. I wonder if instead overflowIndex should only be compared against items.length when items.length is > 0 (leaving the original `items.length - 1` change in.)

Reply:
(I guess some expect error test cases would be useful here too.)